packages/browser/src/sdk.ts

```diff
@@ -16,13 +16,14 @@ import {
 
 import { BrowserClient, BrowserClientOptions, BrowserOptions } from './client';
 import { ReportDialogOptions, WINDOW, wrap as internalWrap } from './helpers';
-import { Breadcrumbs, Dedupe, GlobalHandlers, HttpContext, LinkedErrors, TryCatch } from './integrations';
+import { Breadcrumbs, Dedupe, GlobalHandlers, HttpClient, HttpContext, LinkedErrors, TryCatch } from './integrations';
 import { defaultStackParser } from './stack-parsers';
 import { makeFetchTransport, makeXHRTransport } from './transports';
 
 export const defaultIntegrations = [
   new CoreIntegrations.InboundFilters(),
   new CoreIntegrations.FunctionToString(),
+  new HttpClient(),
```

Comment:
Making this integration part of the default integrations feels a bit weird. Right now it does nothing because `captureFailedRequests` defaults to `false` anyways. Including it here has impact on tree-shakability.

Thoughts @onurtemizkan, @Lms24, @AbhiPrasad?

Reply:
+1 for not making it a default integration for now

Reply:
Yes, makes sense to me 👍 Removed.